test/Tests.Infrastructure/RavenTestBase.cs

```diff
@@ -286,6 +287,9 @@ private DeleteDatabaseResult DeleteDatabase(Options options, RavenServer serverT
             catch (AllTopologyNodesDownException)
             {
 
+            }
+            catch (RavenException ex) when (ex.InnerException is InvalidOperationException)
```

Comment:
why?

Reply:
Because you get failure in tests dispose (when you try to delete the database from a node that doesn't have the db.. the previous code didn't throw because it didn't wait for the RemoveNodeFromDatabasecommands of all the nodes).